zebedee-reader/src/main/java/com/github/onsdigital/zebedee/reader/FileSystemContentReader.java

```diff
@@ -70,6 +76,7 @@ public class FileSystemContentReader implements ContentReader {
 
     private final Path rootFolder;
     protected ContentLanguage language = ContentLanguage.ENGLISH;
+    private Tracer tracer = GlobalOpenTelemetry.getTracer("zebedee", "");
```

Comment:
This should probably be "zebedee-reader" rather than "zebedee" as this sits within the zebedee-reader deployment